workflow/controller/operator.go

```diff
@@ -1645,6 +1654,7 @@ func (woc *wfOperationCtx) initializeNode(nodeName string, nodeType wfv1.NodeTyp
 		BoundaryID:    boundaryID,
 		Phase:         phase,
 		StartedAt:     metav1.Time{Time: time.Now().UTC()},
+		NodeName:      "",
```

Comment:
This is not necessary, omitted `string` fields are automatically created as empty strings.